jdaviz/app.py

```diff
@@ -1014,6 +1014,17 @@ def get_subsets_from_viewer(self, viewer_reference, data_label=None, subset_type
 
         return regions
 
+    def _get_wcs_from_subset(self, subset_state):
+        """ Usually WCS is subset.parent.coords, except special cuebviz case."""
+
+        if self.config == 'cubeviz':
+            parent_data = subset_state.attributes[0].parent
+            wcs = parent_data.meta.get("_orig_spatial_wcs", None)
+        else:
+            wcs = subset_state.xatt.parent.coords
```

Comment:
I know this isn't new code, but maybe we need a tech debt ticket to try to clean this up?